signer/local_cosigner.go

```diff
@@ -284,34 +341,9 @@ func (cosigner *LocalCosigner) dealShares(req CosignerGetEphemeralSecretPartRequ
 
 }
 
-func (cosigner *LocalCosigner) GetEphemeralSecretParts(
-	hrst HRSTKey) (*CosignerEphemeralSecretPartsResponse, error) {
-	res := &CosignerEphemeralSecretPartsResponse{
-		EncryptedSecrets: make([]CosignerEphemeralSecretPart, 0, len(cosigner.peers)-1),
-	}
-	for _, peer := range cosigner.peers {
-		if peer.ID == cosigner.GetID() {
-			continue
-		}
-		secretPart, err := cosigner.getEphemeralSecretPart(CosignerGetEphemeralSecretPartRequest{
-			ID:        peer.ID,
-			Height:    hrst.Height,
-			Round:     hrst.Round,
-			Step:      hrst.Step,
-			Timestamp: time.Unix(0, hrst.Timestamp),
-		})
-
-		if err != nil {
-			return nil, err
-		}
-
-		res.EncryptedSecrets = append(res.EncryptedSecrets, secretPart)
-	}
-	return res, nil
-}
-
 // Get the ephemeral secret part for an ephemeral share
 // The ephemeral secret part is encrypted for the receiver
+// Implements ThresholdEd25519Signature interface
 func (cosigner *LocalCosigner) getEphemeralSecretPart(
```

Comment:
These should move out to a new file for the "local" threshold ed25519 signature implementation.